libs/SalesforceHybrid/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ dependencies {
     api("org.apache.cordova:framework:12.0.1")
     api("androidx.appcompat:appcompat:1.6.1")
     api("androidx.appcompat:appcompat-resources:1.6.1")
-    api("androidx.webkit:webkit:1.8.0")
+    api("androidx.webkit:webkit:1.9.0")
```

Comment:
This is a rider update, but I thought it applicable since we just did a full update to all dependencies for 12.0.0 on `dev`.